SevenSlots/SevenSlots/SevenSlots/ViewModel/BlackjackViewModel.cs

```diff
@@ -1,11 +1,183 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Text;
+using System.Windows.Input;
+using Xamarin.Forms;
 
 namespace SevenSlots.ViewModel
 {
-    public class BlackjackViewModel
+    public class BlackjackViewModel : ViewModelBase
     {
+        #region Constructors...
 
+        public BlackjackViewModel()
+        {
+            TotalBet = 0;
+            OwnedMoney = 12345;
+            TotalWin = 0;
+            InitializeCommands();
+        }
+
+        #endregion
+
+        #region Private Fields...
+
+        private int _totalBet;
+        private int _ownedMoney;
+        private int _totalWin;
+        List<string> _cards = new List<string>()
+        {
+             "c01", "c02", "c03", "c04", "c05", "c06", "c07", "c08", "c09", "c10", "c11", "c12", "d13", "d01", "d02", "d03", "d04", "d05", "d06", "d07", "d08", "d09", "d10", "d11", "d12", "d13", "h01", "h02", "h03", "h04", "h05", "h06", "h07", "h08", "h09", "h10", "h11", "h12", "h13", "s01", "s02", "s03", "s04", "s05", "s06", "s07", "s08", "s09", "s10", "s11", "s12", "s13"
```

Comment:
I suggest that you would put the strings on separate lines like so:
c01, c02...
d01, d02...
and so on